packages/sprinkles/src/types.ts

```diff
@@ -1,3 +1,5 @@
+import type { Condition } from './';
```

Comment:
This is a circular import. Could we move the `Condition` definition in here and import it elsewhere?